data/assetPaths.json

```
@@ -5,4 +5,4 @@
   "app.map": "/assets/js/app-XKFC5Y7M.js.map",
   "styles.css": "/assets/styles/styles-VW6JU45O.css",
   "styles.map": "/assets/styles/styles-VW6JU45O.css.map"
-}
\ No newline at end of file
+}
```

If we add back the newline at the end of the file, it will remove the change entirely from the PR as we want.

```suggestion
}

```

---

nice! ty!